momento-provider/src/test/java/com/google/code/ssm/providers/momento/TestPOC.java

```diff
@@ -99,7 +121,7 @@ public TestObject getComplexObject(@ParameterValueKeyProvider String objectKey)
 
     @Test
     @SneakyThrows
-//    @Ignore // Comment out if you'd like to test this out yourself
+    @Ignore // Comment out if you'd like to test this out yourself
```

Comment:
do we want this committed?

Reply:
One of the tests was marked ignored and I changed this to ignore the other by default as well. Neither of the existing tests write data by default, nor do they have assertions.